scripts/dev/devimint/aliases/fedimint-cli

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$CARGO_BUILD_TARGET_BIN_DIR/$FM_MINT_CLIENT "$@"
```

Comment:
This concatenation is kind of clunky, and generally I think `devimint` should stop setting combined aliases like `FM_MINT_CLIENT`, and only set each thing that the caller can't  know about because `devimint` itself picks them (like data or tmp dirs).

Reply:
Just created #6658 and came up with a more elegant approach.